### PR TITLE
Bug 1849505: Pin Minimist dependecy package to 1.2.5 version

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -239,7 +239,8 @@
   },
   "resolutions": {
     "jquery": "3.5.1",
-    "victory-shared-events": "34.3.8"
+    "victory-shared-events": "34.3.8",
+    "minimist": "1.2.3"
```
Comment (spadgett) [inline frontend/package.json:243]: Any reason not to go to 1.2.5, which is the latest?

File: frontend/package.json [hunk 2]
```diff
@@ -248,5 +249,6 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx,json,gql,graphql}": "eslint --color --fix"
-  }
+  },
+  "version": "0.0.0"
```
Comment (spadgett) [inline frontend/package.json:253]: Was this an intentional change?

Comment (jhadvig) [reply frontend/package.json:253]: no, most probably done by `husky` ?

Comment (jhadvig) [reply frontend/package.json]: but after I updated the minimist to `1.2.3` it's gone... weird

**Comment by jhadvig** [comment]
@spadgett comment addressed.

**Comment by jhadvig** [comment]
@spadgett please take a look again, had to push due to conflict in the `package.json`

**Comment by jhadvig** [comment]
@spadgett shall we fix the minimist version all the way to 4.2 ? I guess it will need to be fixed manually

**Pull request merged**